code/Model/EditableFormField/EditableFileField.php

```diff
@@ -194,7 +194,18 @@ public function validate()
         return $result;
     }
 
+    public function checkIsFolderExist()
+    {
+        return $this->Folder()->ID !== 0 ? true : false;
+    }
 
+    public function setProtectedFolder()
```

Comment:
```suggestion
    public function setProtectedFolder(): void
```
Use method param/return types on new methods, even if it doesn't match the rest of the class